src/idl_gen_python.cpp

```diff
@@ -360,6 +360,39 @@ class PythonGenerator : public BaseGenerator {
     code += Indent + Indent + "return None\n\n";
   }
 
+    // Get the value of a vector's union member.
+    void GetMemberOfVectorOfUnion(const StructDef &struct_def,
+                                 const FieldDef &field,
+                                 std::string *code_ptr) const {
+      auto &code = *code_ptr;
+      auto vectortype = field.value.type.VectorType();
+
+      GenReceiver(struct_def, code_ptr);
+      code += namer_.Method(field);
+      code += "(self, j):" + OffsetPrefix(field);
+      code += Indent + Indent + Indent + "x = self._tab.Vector(o)\n";
+      code += Indent + Indent + Indent;
+      code += "x += flatbuffers.number_types.UOffsetTFlags.py_type(j) * ";
+      code += NumToString(InlineSize(vectortype)) + "\n";
+      code += Indent + Indent + Indent;
+      code += "x -= self._tab.Pos\n";
+
+      // TODO(rw): this works and is not the good way to it:
```

Comment:
I don't think rw is actively working on this stuff anymore, please fix or leave a TODO for yourself

Reply:
On a second look, the `is_native_table` will always be true (since the field is always a union) so I can squash the if statement the TODO seems to be referring to, though it'll remain in the other method.